sql/sql_expression_operations.cc

```diff
@@ -2214,7 +2214,7 @@ bind(SqlBindingScope & scope) const
     BoundFunction fn = scope.doGetFunction(tableName, functionName,
                                            boundArgs, scope);
     
-    if (!fn) {
+    if (!fn && !fn.bindBuiltinFunction) {
```

Comment:
If I throw if fn is false then it will never reach the bindFunction call a couples of lines lower.